workbench/package.json

```diff
@@ -5,14 +5,21 @@
   "main": "build/main/main.js",
   "homepage": "./",
   "scripts": {
-    "start": "electron .",
+    "start": "yarn build-main && yarn build-preload && yarn build-static && concurrently --kill-others \"yarn serve\" \"electron .\"",
+    "serve": "cross-env MODE=development node scripts/watch.js",
     "lint": "eslint --cache --color --ext .jsx,.js src",
-    "test": "jest --runInBand --testPathIgnorePatterns /tests/binary_tests/ /tests/sampledata_linkcheck/",
-    "test-electron-app": "jest --runInBand --testMatch **/tests/binary_tests/puppet.test.js",
-    "test-sampledata-registry": "jest --runInBand --testMatch **/tests/sampledata_linkcheck/sampledata_registry.test.js",
+    "test": "yarn run test-main && yarn run test-renderer && yarn run test-flask",
```

Comment:
I thought the better of this and changed it back. It was a problem that there were multiple summary reports from jest instead of one big summary that is always printed to the console at the end.